src/domain/event/enrol/Enrol.tsx

```diff
@@ -42,14 +46,21 @@ const Enrol: FunctionComponent = () => {
     enrolOccurrenceMutation,
     {
       refetchQueries: [
-        { query: profileQuery },
+        // FIXME: Prevent crash after unenrol:
+        // Rendered fewer hooks than expected. This may be caused by an accidental early return statement.
+        //{ query: profileQuery },
```

Comment:
@jpsalo Do you have any idea why we get this error if we enable the profileQuery refetch?